data/systemd-user/Makefile

```diff
@@ -38,5 +37,4 @@ clean:
 %: %.in
 	cat $< | \
 		sed s:@bindir@:$(BINDIR):g | \
-		sed s:@libexecdir@:$(LIBEXECDIR):g | \
```

Comment:
Same here, has this become used by anything else since the original commit was added?

Reply:
no